.travis.yml

```diff
@@ -194,5 +194,11 @@ matrix:
         - pushd build
         - DESTDIR=./staging ninja install
         - tree ./staging
+
+    # oss-fuzz compilation test
```

Comment:
Just for my understanding :
it's just a quick test which ensures that `oss-fuzz` still compiles correctly,
it doesn't run the test nor deploy anything on Google's oss-fuzz, right ?

I noticed it fails without a "red flag" notification :
https://travis-ci.org/lz4/lz4/jobs/551658718#L455

I suspect that's because `lz4` is not yet part of Google's ossfuzz ?
Also, it means that the test will have to be modified in some way so that it triggers a red flag when it fails ?

Reply:
Yep, you got it :smile: